src/sync/submitters/uniqueKeysSubmitter.ts

```diff
@@ -0,0 +1,37 @@
+import { ISdkFactoryContextSync } from '../../sdkFactory/types';
+import { ISet, setToArray } from '../../utils/lang/sets';
+import { submitterFactory } from './submitter';
+import { UniqueKeysPayload } from './types';
+
+/**
+ * Converts `uniqueKeys` data from cache into request payload.
+ */
+export function fromUniqueKeysCollector(uniqueKeys: { [featureName: string]: ISet<string> }): UniqueKeysPayload {
+  const payload = [];
+  const keys = Object.keys(uniqueKeys);
+  for (let i = 0; i < keys.length; i++) {
+    const featureName = keys[i];
+    const featureKeys = setToArray(uniqueKeys[featureName]);
+    const uniqueKeysPayload = {
+      f: featureName,
+      k: featureKeys
+    };
+
+    payload.push(uniqueKeysPayload);
+  }
+  return payload;
```

Comment:
according to spec, the payload is not an array, it is an object with the format:
```
{ keys: [...]}
```

so, it should be something like: 
```
return { keys: payload };
```